core/lib/dal/src/models/storage_transaction.rs

```diff
@@ -322,6 +322,85 @@ impl From<StorageTransaction> for Transaction {
     }
 }
 
+#[derive(Serialize, Deserialize, sqlx::FromRow)]
+pub struct StorageTransactionReceipt {
+    pub error: Option<String>,
+    pub tx_format: Option<i32>,
+    pub index_in_block: Option<i32>,
+    pub block_hash: Vec<u8>,
+    pub tx_hash: Vec<u8>,
+    pub block_number: i64,
+    pub l1_batch_tx_index: Option<i32>,
+    pub l1_batch_number: Option<i64>,
+    pub transfer_to: Option<sqlx::types::JsonValue>,
+    pub execute_contract_address: Option<sqlx::types::JsonValue>,
+    pub refunded_gas: i64,
+    pub gas_limit: Option<BigDecimal>,
+    pub effective_gas_price: Option<BigDecimal>,
+    pub contract_address: Option<Vec<u8>>,
```

Comment:
Bikeshedding: Maybe, name this field something like `contract_address_storage_key`? Subjectively, "address" suffix implies that it's a 20-byte address, so conversion logic below looks weird.